website/src/controllers/payments/paypalBillingSetup.js

```diff
@@ -1,3 +1,5 @@
+require('babel-core/register');
```

Comment:
That I added because I needed to run that on the command line to generate the Paypal keys and node didn't like the ES6 code.

I'm not 100% sure that's the best way to go about it, but it seemed to work.